staging/src/k8s.io/apiserver/pkg/storage/etcd3/metrics/metrics.go

```diff
@@ -0,0 +1,55 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package etcdmetrics
+
+import (
+	"sync"
+	"time"
+
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+var (
+	etcdRequestLatencies = prometheus.NewHistogramVec(
+		prometheus.HistogramOpts{
+			Name:    "etcd_request_latency_seconds",
```

Comment:
@WIZARD-CXY All looks good, one last thing in mind. This is exactly the same name as https://github.com/kubernetes/kubernetes/blob/24643fd1163857a507e40f4bceb303c5b44d2d8f/staging/src/k8s.io/apiserver/pkg/storage/etcd/metrics/metrics.go#L55, would there be any confusion?

Reply:
Actually the apiserver can use only one etcd backend, either etcd2 or etcd3. I think it is ok.

Reply:
WDYT @xiang90 @sttts

Reply:
Am fine with using the same metrics name. Am surprised that we still have the etcd2 backend.